site/content/tutorial/15-context/01-context-api/text.md

```diff
@@ -46,4 +45,12 @@ const key = {};
 
 We can use anything as a key — we could do `setContext('mapbox', ...)` for example. The downside of using a string is that different component libraries might accidentally use the same one; using an object literal means the keys are guaranteed not to conflict in any circumstance, even when you have multiple different contexts operating across many component layers.
 
-> Remember that context is not inherently reactive. If you need context values to be reactive then you can pass a store into context, which *will* be reactive.
+## Contexts vs. stores
+
+Contexts and stores seem similar. They differ in that stores are available to *any* part of an app, while a context is only *available to a component and its descendants*. This can be helpful if you want to use several instances of a component without the state of one interfering with the state of the others.
```

Comment:
This sounds good, but it's gonna bug me that the 'available to' is not italicized the first time it appears in the sentence but it is the second time in a parallel construction.

Reply:
oof. mad that i missed that 😅